db/seeds.rb

```diff
@@ -11,13 +11,16 @@
   Employee.create!(email: "employee#{i}@test.com", password: 'password')
 end
 
-Kudo.create!(title: 'From console', content: 'Something', giver: Employee.find_by(email: 'employee1@test.com'), receiver: Employee.find_by(email: 'employee2@test.com'))
+CompanyValue.create!(title: 'Honesty')
```

Comment:
let's try make it a bit more 'ruby' way
```suggestion
%w[Honesty Ownership Accountability Passion].each do |company_value_title|
  CompanyValue.create!(title: company_value_title)
end
```

Reply:
I corrected it. Thank you for your advice.